client/packages/system/src/Patient/DocumentHistory/DocumentHistory.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Typography,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+} from '@mui/material';
+import { ChevronDownIcon } from '@common/icons';
+import { DocumentNode } from 'packages/common/src/types/schema';
+import React, { FC, useEffect, useState } from 'react';
+import ReactDiffViewer from 'react-diff-viewer';
+import { usePatient } from '../api';
+import { Box } from 'packages/common/src';
+
+export const DocumentHistory: FC<{ documentName: string }> = ({
+  documentName,
+}) => {
+  const [history, setHistory] = useState([] as DocumentNode[]);
+
+  const { data: docHistory } = usePatient.document.history(documentName);
+  useEffect(() => {
+    if (docHistory) {
+      setHistory(docHistory);
+    }
+  }, [docHistory]);
+
+  const findFirstParent = (
+    head: DocumentNode,
+    history: DocumentNode[]
+  ): DocumentNode | undefined => {
+    const parentId = head.parents[0];
+    if (parentId === undefined) {
+      return undefined;
+    }
+    return history.find(node => node.id === parentId);
+  };
+  const toString = (node: DocumentNode | undefined): string => {
+    if (node === undefined) {
+      return '';
+    }
+    const clone = JSON.parse(JSON.stringify(node));
+    delete clone['__typename'];
```

Comment:
does this create empty keys "" ?

Reply:
It removes the property completely so that it isn't shown at allow